apps/src/lab2/views/components/NoExemplarPage.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import moduleStyles from './no-exemplar-page.module.scss';
+
+const NoExemplarPage: React.FunctionComponent = () => {
+  return (
+    <div className={moduleStyles.noExemplarContainer}>
+      No exemplar exists for this level.
+    </div>
```

Comment:
How would a user potentially navigate to this page?

Reply:
They could either add `?exemplar=true` to the current level url or if they are already on an exemplar level and click a bubble in their current progression for a level that doesn't have an exemplar.

Reply:
That makes sense. Does this style of exemplar link also open in a new tab? I'm more familiar with the project link style, but I'm trying to understand how a user would get back to the standard view.

Reply:
Yeah it opens in a new tab.

Reply:
Thanks. I had been wondering if we should consider adding a link her to take teachers back to the level view (or to take levelbuilders to the exemplar edit view), but given that our current flow is to close the tab, that doesn't feel as necessary.